apiserver/facades/client/modelmanager/modelmanager.go

```diff
@@ -806,55 +806,67 @@ func (m *ModelManagerAPI) ListModelSummaries(ctx context.Context, req params.Mod
 		return result, errors.Trace(err)
 	}
 
-	modelInfos, err := m.state.ModelSummariesForUser(userTag, req.All && m.isAdmin)
+	var modelInfos []model.ModelSummary
```

Comment:
I think a model summary should live in `core/model`

Reply:
Moved.